tutorials/run-laravel-on-appengine-standard/index.md

```diff
@@ -165,7 +165,10 @@ Laravel, you need to manually add the `DB_SOCKET` value to
           DB_DATABASE: YOUR_DB_DATABASE
           DB_USERNAME: YOUR_DB_USERNAME
           DB_PASSWORD: YOUR_DB_PASSWORD
+          ## for MYSQL, use DB_SOCKET:
           DB_SOCKET: "/cloudsql/YOUR_CONNECTION_NAME"
+          ## for PostgreSQL, use DB_HOST:
+          DB_HOST: "/cloudsql/YOUR_CONNECTION_NAME"
```

Comment:
Leave this one commented out so the default is mysql

Reply:
makes sense to me, thanks for taking a look

Reply:
@tjwebb , I'll merge this as soon as you make the recommended change.